doc/dev/BUILD_WASM.md

```diff
@@ -0,0 +1,111 @@
+# Introduction
+
+F3D can be built in WebAssembly using emscripten in order to embed it into a web browser.
+It is still experimental and only a small fraction of the libf3d public API is bound.
+An example can be seen at https://f3d.app/web
+
+This guide is describing how to build VTK and F3D with emscripten using Docker on Linux.
+
+# Building
+
+While it's possible to setup an emscripten cross-compiling toolchain locally, it's easier to use Docker to build it
+
+## Preparing the build
+
+Install Docker locally and pull the WebAssembly image
+
+```sh
+docker pull dockcross/web-wasm:20240529-0dade71
+```
+
+Clone VTK and F3D, it will be assumed that the source code is located in `$VTK_DIR` and `$F3D_DIR` variables in the next steps of this guide.
```

Comment:
State needed version of VTK ?

Reply:
let's specify it when Jaswant's MR is merged and the patch in the CI is not needed anymore

Reply:
fair enough